dev-environment/nussknacker/opt/prinz-sample/base-application.conf

```diff
@@ -0,0 +1,82 @@
+storageDir: ./storage
```

Comment:
Directory `/nussknacker-dist/src/universal/conf/` which contains configs is not available in the image.